src/OpenDiffix.Core/CommonTypes.fs

```diff
@@ -205,7 +205,7 @@ type Plan =
 // Executor
 // ----------------------------------------------------------------
 
-type AnonymizationContext = { BucketSeed: Hash }
+type AnonymizationContext = { BucketSeed: Hash; BaseLabels: Value list }
```

Comment:
Are BaseLabels supposed to be implicit values of grouping labels inferred by WHERE `x=const`?

Reply:
Yes. They are labels that implicitly apply to the entire anonymized result set.